### PR TITLE
Create progress bar wrapper to use with dask

### PR DESCRIPTION
**Pull request by aplavin**
Implements #278 as a submodule, respecting dask API for progress bars.

Dask API:

```
from dask.diagnostics import ProgressBar

with ProgressBar():
    ... .compute()
```

This submodule:

```
from tqdm.dask import ProgressBar

with ProgressBar():
    ... .compute()
```

### CODE REVIEW

**Comment by lrq3000** [comment]
TODO (not necessarily by you @aplavin!):
- [ ] Should rename ProgressBar() to tqdm_dask() to respect the nomenclatura of other submodules (and the brand name ;) ).
- [ ] Fix unit test failing.
- [ ] Flake8 check.
- [ ] Add to README.

File: tqdm/__init__.py [hunk 1]
```diff
@@ -26,3 +26,9 @@ def tnrange(*args, **kwargs):  # pragma: no cover
     """
     from ._tqdm_notebook import tnrange as _tnrange
     return _tnrange(*args, **kwargs)
+
+
+try:
+    import ._dask as dask
```
Comment (CrazyPython) [inline tqdm/__init__.py]: This is not valid Python 2.

File: tqdm/_dask.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        """
+        Progress bar class to use with dask.
+        
+        Parameters
+        ----------
+        tclass  : class or instance of tqdm (e.g. tqdm_notebook())
+        targs, tkwargs  : arguments for the tqdm instance
+        """
+        if not isinstance(tclass, type):
+            tclass = type(class)
```
Comment (CrazyPython) [inline tqdm/_dask.py]: `class` -> `tclass`

**Comment by IvoMerchiers** [comment]
Any progress on this?

**Comment by casperdcl** [comment]
@IvoMerchiers thanks for bumping this; probably will be merged within a week

**Comment by lpsinger** [comment]
I see this has been sitting for several months. Any updates?

**Pull request closed without merging**